pkg/xds/experimental/client/gcp/helpers.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package gcp
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+
+	corepb "github.com/cilium/proxy/go/envoy/config/core/v3"
+)
+
+func Node(projectNum int64, mesh string, locality *corepb.Locality, clientID, userAgent string) *corepb.Node {
```

Comment:
please add comments to all exported functions

Reply:
Removed GCP-specific code.